app/controllers/questions_controller.rb

```diff
@@ -1,31 +1,42 @@
 class QuestionsController < ApplicationController
-  before_action :find_question, only: %i(show destroy)
-  before_action :find_test, only: %i(index create)
+  before_action :find_question, only: %i(show destroy edit update)
+  before_action :find_test, only: %i(index create new edit show destroy)
   rescue_from ActiveRecord::RecordNotFound, with: :rescue_with_question_not_found
 
   def index
-    render plain: @test.questions.pluck(:body)
+    @questions = @test.questions
   end
 
   def show
-    render plain: @question.body
+  end
+
+  def edit
   end
 
   def new
+    @question = Question.new
   end
 
   def create
-    question = @test.questions.create(question_params)
-    if question.save
-      render plain: question.inspect
+    @question = @test.questions.new(question_params)
+    if @question.save
+      redirect_to @question
+    else
+      render :new
+    end
+  end
+
+  def update
+    if @question.update(question_params)
+      redirect_to @question
     else
-      render plain: question.errors
+      render :new
     end
   end
 
   def destroy
     @question.destroy
-    render plain: "Destroy question with id=#{@question.id}"
+    redirect_to test_questions_path(@test)
```

Comment:
```suggestion
    redirect_to @question.test
```

После удаления объекта из базы он остаётся в памяти. Хотя его свойста нельзя изменять поскольку он становится `frozen?`.